jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFSharedTldsProcessor.java

```diff
@@ -67,7 +68,7 @@ private Map<String, List<TldMetaData>> getMap() {
 
     private TldMetaData parseTLD(InputStream is) throws Exception {
         try {
-            final XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            final XMLInputFactory inputFactory = XMLInputFactoryUtil.create();
```

Comment:
We must not have tests that exercise this in the testsuite because the org.jboss.as.jsf module has no dependency on org.wildfly.common. So this would not work.

So that dep needs to be added.